tensorflow/compiler/tf2tensorrt/convert/utils.cc

```diff
@@ -56,6 +56,20 @@ Status TrtPrecisionModeFromName(const string& name, TrtPrecisionMode* mode) {
 
 #if GOOGLE_CUDA && GOOGLE_TENSORRT
 
+Status ConcatenateShape(nvinfer1::INetworkDefinition* network, string node_name,
+                        const std::vector<nvinfer1::ITensor*> size_tensors,
+                        nvinfer1::ITensor** new_shape) {
+  nvinfer1::IConcatenationLayer* layer = network->addConcatenation(
+      const_cast<nvinfer1::ITensor* const*>(size_tensors.data()),
+      size_tensors.size());
```

Comment:
We need to add a layer name to support xprof.

Reply:
I have added the name. Moved the util to convert_nodes.cc to have access to Converter::SetLayerName